config/defaults.json

```diff
@@ -0,0 +1,135 @@
+{
+  "cli_wallet_connection": "",
+  "db": {
```

Comment:
Yeah I figure fewer developer deps is a win. I think we should update your raw queries to use orm methods though for max compatibility.